### ISSUE TITLE
Boolean literals true and false

### ISSUE THREAD

**Issue by Clashsoft** [fujaba/fulibScenarios#219]
Currently, there is no way to produce a boolean value like `true` or `false`.

If a boolean attribute is desired, there is a workaround:

```
Every LED has a status of type boolean.
```

What we really want:

```
There is a LED with status true.
```

Or better:

```
There is a LED with status ON.
```

**Comment by Clashsoft** [comment]
Not sure what keywords to use for this. Possible options:

- `true` / `false`
- `yes` / `no`
- `on` / `off`
- ...?

The right choice really depends on the context. Maybe all?

@azuendorf, what do you think?

**Comment by Clashsoft** [comment]
We are using `true` and `false` for now, implemented in #224. The other aliases can be used by declaring them as variables:

```
We write true into ON.
We write false into OFF.
```
